sa/database.go

```diff
@@ -52,18 +72,45 @@ func NewDbMap(dbConnect string) (*gorp.DbMap, error) {
 	return dbmap, err
 }
 
-// recombineURLForDB transforms a database URL to a URL-like string
-// that the mysql driver can use. The mysql driver needs the Host data
-// to be wrapped in "tcp()" but url.Parse will escape the parentheses
-// and the mysql driver doesn't understand them. So, we can't have
-// "tcp()" in the configs, but can't leave it out before passing it to
-// the mysql driver. Similarly, the driver needs the password and
-// username unescaped. Compromise by doing the leg work if the config
-// says the database URL's scheme is a fake one called
-// "mysql+tcp://". See
-// https://github.com/go-sql-driver/mysql/issues/362 for why we have
-// to futz around and avoid URL.String.
-func recombineURLForDB(dbConnect string) (string, error) {
+// adjustMysqlConfig sets certain flags that we want on every connection.
+func adjustMysqlConfig(conf *mysql.Config) *mysql.Config {
+
```

Comment:
Replace this blank linke with `// Required to turn DATETIME fields into time.Time`